polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -936,4 +973,22 @@ mod tests {
 		};
 		assert!(ForeignChainAliasAccount::<[u8; 32]>::convert_location(&mul).is_none());
 	}
+
+	#[test]
+	fn remote_account_convert_on_para_sending_relay_treasury() {
+		let mul = MultiLocation {
```

Comment:
just a very nit, I would rename `let mul` to `let loc` or `let location`. E.g. [xcm-v4](https://github.com/paritytech/polkadot-sdk/pull/1230) renames `MultiLocation` to `Location`, so `let mul = Location...` will look strange

Reply:
resolved